sound/soc/sdca/sdca_functions.c

```diff
@@ -191,6 +192,18 @@ static int find_sdca_function(struct acpi_device *adev, void *data)
 	if (!valid_function_found)
 		return 0;
 
+	fwnode_property_read_u32(function_node, "mipi-sdca-function-busy-max-delay",
+				 &max_busy_delay);
+	sdca->functions[sdca->function_count].function_busy_max_delay_us =
+		max_busy_delay;
+	if (max_busy_delay > 0) {
+		dev_err(parent,
+			"%pfwP: unsupported non-zero delay %d us for Function_Busy\n",
+			function_node, max_busy_delay);
+		dev_err(parent,
+			"please report this on https://github.com/thesofproject/linux/issues\n");
```

Comment:
Well you say there is no hardware that uses this feature, but cs42l43 and likely its successors make heavy use of it if they are used in SDCA mode. Not that support needs to be added right now, but just making you aware.